README.md

```diff
@@ -142,6 +142,16 @@ To update the Design Switcher, temporarily set the `DISABLE_LIVE_MODE` flag in `
 
 The Tutorial's DesignDoc is set to the main development Figma file, to assist in development of the app, but the committed serialized file is a separate file that presents a "welcome" page. This file's ID is `BX9UyUa5lkuSP3dEnqBdJf`. To update the file, fetch the `BX9UyUa5lkuSP3dEnqBdJf` file, then replace `reference-apps/tutorial/app/src/main/assets/figma/TutorialDoc_3z4xExq0INrL9vxPhj9tl7` with the serialized file. **The file name will remain the same**, the `TutorialDoc_3z4xExq0INrL9vxPhj9tl7` fill will contain the serialized `BX9UyUa5lkuSP3dEnqBdJf` file. The Tutorial app project has an AndroidIntegratedTest to ensure that the correct file is set, and it will be run as part of running the `./dev-scripts/test-all.sh` script.
 
+## Roborazzi screenshot tests
+
+[Roborazzi](https://github.com/takahirom/roborazzi) is a new framework that allows for screenshot testing of Android Apps on your local system. It uses [Robolectric](https://github.com/robolectric/robolectric), the standard unit testing framework for Android, to render DesignCompose locally, allowing screenshots to be generated. The screenshots won't be one-to-one with actual Android devices, but they'll be very close and stable enough for changes to be detected.
```

Comment:
As a side note, it appears that Robolectric Native Graphics currently doesn't work on Windows. Because of this, Now In Android has implemented a mechanism to update images on CI. For more information, you can refer to these links:
- [Robolectric Issue](https://github.com/robolectric/robolectric/issues/8312)
- [Now In Android Screenshot Tests](https://github.com/android/nowinandroid#screenshot-tests)"

Reply:
Thanks for the heads up! That reminds me to take anotherlook at NiA and see what ideas I can crib from them.